src/Marten.NodaTime/Marten.NodaTime.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
     <PropertyGroup>
         <Description>NodaTime extension for Marten</Description>
-        <VersionPrefix>6.0.0-alpha.3</VersionPrefix>
+        <VersionPrefix>6.0.0-alpha.4</VersionPrefix>
```

Comment:
It's already set in the `Directory.Build.props`, so could be removed here as that value will be taken over.

Reply:
Thanks for the suggestion. For now we decided to keep it also here to be able to release it independently.